Package.swift

```diff
@@ -44,8 +44,24 @@ let package = Package(
             name: "SpeziLLMLocal",
             dependencies: [
                 .target(name: "SpeziLLM"),
+                .target(name: "SpeziLLMLocalHelpers"),
                 .product(name: "llama", package: "llama.cpp"),
                 .product(name: "Spezi", package: "Spezi")
+            ],
+            swiftSettings: [
+                .interoperabilityMode(.Cxx)
+            ]
+        ),
+        .target(
+            name: "SpeziLLMLocalHelpers",
+            dependencies: [
+                .product(name: "llama", package: "llama.cpp")
+            ],
+            cxxSettings: [
+                .unsafeFlags(["-std=c++11"])
```

Comment:
Upon closer examination, this might really be an issue, even for an internal target that isn't exposed as a `library`.
Will investigate further tomorrow, sadly, the C++11 standard is a requirement here

Reply:
Yep, just tried that, sadly doesn't work even though the target is internal :(

Reply:
Solved via https://github.com/StanfordBDHG/llama.cpp/pull/8